docker/docker-compose.yml

```diff
@@ -362,11 +358,13 @@ services:
       - /bin/bash
       - -c
       - |
+        apt-get update && apt-get install -y gettext-base
+        cd /data/
         set -e
         envsubst < config.toml.in > config.toml
-        bitcoin-cli -rpcwait -rpcconnect=bitcoin getmininginfo
```

Comment:
IIUC this was used to ensure the bitcoin node was in a happy state. Is the stacks node going to complain if it goes up when the bitcoin node is still warming up?

Reply:
Oh yeah, it might. We can add in a docker compose `depends_on` to make sure that bitcoin core is healthy before starting the stacks node.